src/TvMaze.Api.Client/Configuration/RetryRateLimitingStrategy.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Polly;
+
+namespace TvMaze.Api.Client.Configuration
+{
+    /// <summary>
+    /// This implementation will retry the request multiple times, if a rate limit is encountered.
+    /// </summary>
+    public class RetryRateLimitingStrategy : IRateLimitingStrategy
+    {
+        private readonly AsyncPolicy<HttpResponseMessage> _policy;
+
+        /// <summary>
+        /// Creates an instance with reasonable default values for the rate limits described by the API.
+        /// </summary>
+        public RetryRateLimitingStrategy()
+            : this(5, TimeSpan.FromSeconds(3))
```

Comment:
Can you move these default values to constants so we know what the numbers mean? Something like `DefaultNumberOfRetries` and `DefaultRetryInterval`.